app/src/main/java/com/hedvig/app/HedvigApplication.kt

```diff
@@ -67,7 +70,9 @@ open class HedvigApplication : Application() {
         if (authenticationTokenService.authenticationToken == null) {
             whatsNewRepository.removeNewsForNewUser()
             CoroutineScope(IO).launch {
-                acquireHedvigToken()
+                if (!featureManager.isFeatureEnabled(Feature.QUOTE_CART)) {
```

Comment:
Hmm what's the interaction like about this where we don't want this token when using quote cart? This is handled differntly in that case I presume? is the `authenticationTokenService` also updated in that approach, or something completely different?

Reply:
Good question. I will ask the backenders!